torch_geometric/loader/neighbor_loader.py

```diff
@@ -207,6 +225,7 @@ def __init__(
         directed: bool = True,
         transform: Callable = None,
         neighbor_sampler: Optional[NeighborSampler] = None,
+        node_time: Optional[Dict[NodeType, Tensor]] = None,
```

Comment:
Can we add a docstring about the new functionality?

Reply:
done.